src/profile-logic/transforms.js

```diff
@@ -189,9 +232,20 @@ export function stringifyTransforms(transforms: TransformStack = []): string {
   return transforms
     .map(transform => {
       const shortKey = TRANSFORM_TO_SHORT_KEY[transform.type];
+      if (!shortKey) {
+        throw new Error(
+          'Expected to be able to convert a transform into its short key.'
+        );
+      }
+      // This switch breaks down each transform down into shared groups of what data
```

Comment:
(sorry if this is a dupe, but I don't see it so writing it again)
nit: `down` is repeated, I believe the first one should be removed.